jbpm-console-ng-workbench-integration/jbpm-console-ng-workbench-integration-api/src/main/java/org/jbpm/console/ng/wi/dd/model/RuntimeStrategy.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.wi.dd.model;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+@Portable
+public enum RuntimeStrategy {
```

Comment:
How about reusing the runtimestrategy and perhaps even translations from the server management module?